tests/pr_test.go

```diff
@@ -8,24 +8,29 @@ import (
 	"github.com/terraform-ibm-modules/ibmcloud-terratest-wrapper/testhelper"
 )
 
-// Use existing resource group
-const resourceGroup = "geretain-test-resources"
 const completeExampleDir = "examples/complete"
 
+// Use existing group for tests
+const resourceGroup = "geretain-test-event-notifications"
+
+// Hardcoding the region value here as EN only supports: us-south (Dallas), eu-gb (London), eu-de (Frankfurt), au-syd (Sydney)
+const region = "us-south"
```

Comment:
you could randomize the region selection instead of always using us-south. See https://github.com/terraform-ibm-modules/terraform-ibm-app-configuration/blob/96decc8013c768b7781362d03a75e791ee901c27/tests/pr_test.go#L18-L23

Reply:
Done.